packages/wallets/src/features/cashier/modules/TransactionStatus/components/TransactionStatusSuccess/__tests__/TransactionStatusSuccess.spec.tsx

```diff
@@ -0,0 +1,229 @@
+import { APIProvider } from '@deriv/api';
+import { fireEvent, render, screen } from '@testing-library/react';
+import { ModalProvider } from '../../../../../../../components/ModalProvider';
+import TransactionStatusSuccess from '../TransactionStatusSuccess';
+
+jest.mock('react-router-dom', () => ({
+    useHistory: jest.fn(),
+}));
+const mockUseHistory = useHistory as jest.Mock;
+
+const mockTransactions = [
+    {
+        address_hash: '',
+        address_url: '',
+        amount: 0.0001,
+        description: '',
+        formatted_address_hash: '',
+        formatted_amount: '',
+        formatted_confirmations: '',
+        formatted_transaction_hash: '',
+        id: '',
+        is_deposit: false,
+        is_valid_to_cancel: 1 as const,
+        is_withdrawal: true,
+        status_code: 'LOCKED' as const,
+        status_message: '',
+        status_name: '',
+        submit_date: 123456,
+        transaction_type: 'withdrawal' as const,
+    },
+];
+
+const mockWallet = {
+    account_category: 'wallet' as const,
+    account_type: '',
+    balance: 1,
+    broker: '',
+    created_at: new Date('01/01/1970'),
+    currency: '',
+    currency_config: {
+        code: '',
+        display_code: '',
+        fractional_digits: 1,
+        is_AUD: false,
+        is_BTC: true,
+        is_BUSD: false,
+        is_crypto: true,
+        is_DAI: false,
+        is_deposit_suspended: 0 as const,
+        is_ETH: false,
+        is_EUR: false,
+        is_EURS: false,
+        is_eUSDT: false,
+        is_fiat: false,
+        is_GBP: false,
+        is_IDK: false,
+        is_LTC: false,
+        is_PAX: false,
+        is_suspended: 0 as const,
+        is_TUSD: false,
+        is_tUSDT: false,
+        is_USB: false,
+        is_USD: false,
+        is_USDC: false,
+        is_USDK: false,
+        is_USDT: false,
+        is_withdrawal_suspended: 0 as const,
+        minimum_withdrawal: 0.1,
+        name: '',
+        stake_default: 0.1,
+        transfer_between_accounts: {
+            fees: {},
+            limits: { max: 0.1, min: 0.1 },
+            limits_ctrader: { max: 0.1, min: 0.1 },
+            limits_derivez: { max: 0.1, min: 0.1 },
+            limits_dxtrade: { max: 0.1, min: 0.1 },
+            limits_mt5: { max: 0.1, min: 0.1 },
+        },
+        type: 'crypto' as const,
+    },
+    display_balance: '',
+    dtrade_loginid: undefined,
+    excluded_until: undefined,
+    is_active: true,
+    is_crypto: true,
+    is_disabled: false,
+    is_malta_wallet: false,
+    is_mf: false,
+    is_trading: false,
+    is_virtual: false,
+    is_wallet: true,
+    landing_company_name: '',
+    linked_to: [],
+    loginid: '',
+    platform: 'deriv' as const,
+    wallet_currency_type: '',
+};
+
+describe('TransactionStatusSuccess', () => {
+    beforeEach(() => {
+        jest.clearAllMocks();
+    });
+
+    it('should render correct elements for withdrawal transactions', () => {
```

Comment:
for future PRs: better to have more specific test cases, 
e.g. shoud display properly formatted transation value
should display "View more" button
should display "No recent transactions" when  no transactions available,

that way - I know immediately which feature I have broken and what features are actually important,

Reply:
Will keep that in mind for future PRs 👍 as per discussion the title for these unit tests has also been updated to be more specific for the different scenarios